### ISSUE TITLE
Use rdfType for table schema terms borrowed from DwC

### ISSUE THREAD

**Issue by dshorthouse** [tdwg/camtrap-dp#167]
Could you use DwC term URIs as values for rdfType elements in your table schema, re-use their definitions for the descriptions & likewise structure the terms in the familiar DwC CamelCase, all in the name of minimizing interpretation?

```
    {
      "name": "scientificName",
      "type": "string",
      "rdfType": "http://rs.tdwg.org/dwc/terms/scientificName",
      "format": "default",
      "description": "The full scientific name, with authorship and date information if known. When forming part of an Identification, this should be the name in lowest level taxonomic rank that can be determined. This term should not contain identification qualifications, which should instead be supplied in the IdentificationQualifier term.",
      "example": "Canis lupus",
      "constraints": {
        "required": false
      }
    }
```

**Comment by peterdesmet** [comment]
Thanks for the suggestion @dshorthouse! You list three action items:

- [ ] Link to corresponding Darwin Core term using an `rdfType` link. I think that is an excellent idea.
- [ ] Update the definitions to align with the Darwin Core definitions. I would rather not copy them verbatim, since Darwin Core definitions often refer to other terms (e.g. here `Identification` and `IdentificationQualifier`). I prefer to borrow for the definitions, but make them understandable within the context of Camtrap DP. Darwin Core Archive elements (e.g. https://rs.gbif.org/core/dwc_taxon.xml) do the same.
- [ ] Use  `lowerCamelCase` for all properties, rather than `snake_case`. There isn't really something that prevents us from doing so. All properties from Data Package and Data resource are single lowercase words. Table Schema and CSV Dialect already use `lowerCamelCase`. It might be good to change and align with these _and_ Darwin Core

@ben-norton @kbubnicki thoughts?

**Comment by ben-norton** [comment]
1. I agree
2. I completely agree. The task is a balancing act between defining the term within the context of Camtrap-DP while maintaining a clear association with the current DWC definition. 
3. To maintain consistency, the DWC naming conventions should be adhered to in Camtrap-DP.

**Comment by peterdesmet** [comment]
This topic was discussed in #169 and implemented in #171. We used `skos` terms over rdfType.
